### PR TITLE
Update block device support for 1.11

### PR DESCRIPTION
**Pull request by jsafrane**
Fixes: https://github.com/kubernetes/website/issues/8810

cc: @msau42

### CODE REVIEW

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -499,7 +498,21 @@ spec:
 
 ## Raw Block Volume Support
 
-Static provisioning support for Raw Block Volumes is included as an alpha feature for v1.9. With this change are some new API fields that need to be used to facilitate this functionality. Kubernetes v1.10 supports only Fibre Channel and Local Volume plugins for this feature.
+The following volume plugins support raw block volumes incl. dynamic provisioning where applicable:
```
Comment (msau42) [inline content/en/docs/concepts/storage/persistent-volumes.md]: including

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -272,10 +270,11 @@ Currently, storage size is the only resource that can be set or requested.  Futu
 
 ### Volume Mode
 
-Prior to v1.9, the default behavior for all volume plugins was to create a filesystem on the persistent volume. With v1.9, the user can specify a `volumeMode` which will now support raw block devices in addition to file systems. Valid values for `volumeMode` are "Filesystem" or "Block". If left unspecified, `volumeMode` defaults to "Filesystem" internally. This is an optional API parameter. 
+Prior to v1.9, the default behavior for all volume plugins was to create a filesystem on the persistent volume. With v1.9, the user can specify a `volumeMode` which will now support raw block devices in addition to file systems. Valid values for `volumeMode` are "Filesystem" or "Block". If left unspecified, `volumeMode` defaults to "Filesystem" internally. This is an optional API parameter.
+
 
 {{< note >}}
-**Note:** This feature is alpha in v1.9 and may change in the future. 
+**Note:** This feature is alpha in v1.9 and may change in the future. Alpha feature `BlockVolume` must be enabled in all cluster components for raw block storage to work.
```
Comment (msau42) [inline content/en/docs/concepts/storage/persistent-volumes.md]: I think we use this tag now to indicate feature status:
```
{{< feature-state for_k8s_version="v1.9" state="alpha" >}}
```

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 3]
```diff
@@ -499,7 +498,21 @@ spec:
...
+* FC (Fibre Channel)
+* GCEPersistentDisk
+* iSCSI
+* Local volume
+* RBD (Ceph Block Device)
+
+
+
+{{< note >}}
+**Note:** This feature is alpha in v1.9 and may change in the future. Alpha feature `BlockVolume` must be enabled in all cluster components for raw block storage to work.
```
Comment (msau42) [inline content/en/docs/concepts/storage/persistent-volumes.md]: Same thing, feature tag

**Comment by msau42** [comment]
@mistyhacks I think this PR is to update documentation as of 1.10

**Comment by msau42** [comment]
Actually @jsafrane is the dynamic provisioning fix going to be in 1.10 too?

**Comment by jsafrane** [comment]
> @mistyhacks I think this PR is to update documentation as of 1.10

No, I target 1.11. I reused old branch name :-)

> Actually @jsafrane is the dynamic provisioning fix going to be in 1.10 too?

No, I think nobody complained about it so alpha users are probably happy with block volumes.

**Comment by jsafrane** [comment]
Rebased

File: content/en/docs/concepts/storage/persistent-volumes.md (hunk 2 above)
Comment (jsafrane) [reply content/en/docs/concepts/storage/persistent-volumes.md]: Fixed

File: content/en/docs/concepts/storage/persistent-volumes.md (hunk 3 above)
Comment (jsafrane) [reply content/en/docs/concepts/storage/persistent-volumes.md]: Fixed

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 4]
```diff
@@ -499,7 +497,19 @@ spec:
...
-Static provisioning support for Raw Block Volumes is included as an alpha feature for v1.9. With this change are some new API fields that need to be used to facilitate this functionality. Kubernetes v1.10 supports only Fibre Channel and Local Volume plugins for this feature.
+{{< feature-state for_k8s_version="v1.9" state="alpha" >}}
+
+To enable this feature in v1.9, the cluster administrator needs to enable the `BlockVolume` feature gate on the apiserver, controller-manager and the kubelet.
+
+The following volume plugins support raw block volumes including dynamic provisioning where applicable:
+
+* AWSElasticBlockStore
+* AzureDisk
+* FC (Fibre Channel)
```
Comment (msau42) [inline content/en/docs/concepts/storage/persistent-volumes.md]: Only FC and iSCSI were implemented in 1.9?  The rest were 1.10

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 5]
```diff
@@ -499,18 +502,25 @@ spec:
...
+apiserver, controller-manager and the kubelet.
 
-The following volume plugins support raw block volumes including dynamic provisioning where applicable:
+The following volume plugins support raw block volumes, including dynamic provisioning where
+applicable.
 
 * AWSElasticBlockStore
 * AzureDisk
-* FC (Fibre Channel)
+* FC (Fiber Channel)
```
Comment (msau42) [inline content/en/docs/concepts/storage/persistent-volumes.md]: It is actually FibreChannel

Comment (mistyhacks) [reply content/en/docs/concepts/storage/persistent-volumes.md]: Wow #TIL! I will raise a follow-on PR for this.

**Comment by mistyhacks** [comment]
@msau42 Is this PR still valid since k/features 351 is slipping to 1.12? Let me know ASAP if we need to roll it back, please.

**Comment by msau42** [comment]
It's still valid. We missed some documentation in 1.10 and fixed some bugs in 1.11.  This is the current state as of 1.11.

**Pull request merged**